easybuild/easyconfigs/r/R/R-4.3.2-gfbf-2023b.eb

```diff
@@ -0,0 +1,375 @@
+name = 'R'
+version = '4.3.2'
+
+homepage = 'https://www.r-project.org/'
+description = """R is a free software environment for statistical computing
+ and graphics."""
+
+toolchain = {'name': 'gfbf', 'version': '2023b'}
+
+source_urls = ['https://cloud.r-project.org/src/base/R-%(version_major)s']
+sources = [SOURCE_TAR_GZ]
+checksums = ['b3f5760ac2eee8026a3f0eefcb25b47723d978038eee8e844762094c860c452a']
+
+builddependencies = [
+    ('pkgconf', '2.0.3'),
+    ('Autotools', '20220317'),
+]
+dependencies = [
+    ('X11', '20231019'),
+    ('Mesa', '23.1.9'),
+    ('libGLU', '9.0.3'),
+    ('cairo', '1.18.0'),
+    ('libreadline', '8.2'),
+    ('ncurses', '6.4'),
+    ('bzip2', '1.0.8'),
+    ('XZ', '5.4.4'),
+    ('zlib', '1.2.13'),
+    ('SQLite', '3.43.1'),
+    ('PCRE2', '10.42'),
+    ('libpng', '1.6.40'),  # for plotting in R
+    ('libjpeg-turbo', '3.0.1'),  # for plottting in R
+    ('LibTIFF', '4.6.0'),
+    ('Java', '11', '', SYSTEM),
+    ('libgit2', '1.7.2'),
+    ('OpenSSL', '1.1', '', SYSTEM),
+    ('cURL', '8.3.0'),
+    ('Tk', '8.6.13'),  # for tcltk
+    ('HarfBuzz', '8.2.2'),  # for textshaping
+    ('FriBidi', '1.0.13'),  # for textshaping
+
+]
+
+# Some R extensions (mclust, quantreg, waveslim for example) require the math library (-lm) to avoid undefined symbols.
+# Adding it to FLIBS makes sure it is present when needed.
+preconfigopts = 'export FLIBS="$FLIBS -lm" && '
+
+configopts = "--with-pic --enable-threads --enable-R-shlib"
+# some recommended packages may fail in a parallel build (e.g. Matrix), and
+# we're installing them anyway below
+configopts += " --with-recommended-packages=no"
+
+# specify that at least EasyBuild v3.5.0 is required,
+# since we rely on the updated easyblock for R to configure correctly w.r.t. BLAS/LAPACK
+easybuild_version = '3.5.0'
+
+exts_default_options = {
+    'source_urls': [
+        'https://cran.r-project.org/src/contrib/Archive/%(name)s',  # package archive
+        'https://cran.r-project.org/src/contrib/',  # current version of packages
+        'https://cran.freestatistics.org/src/contrib',  # mirror alternative for current packages
+    ],
+    'source_tmpl': '%(name)s_%(version)s.tar.gz',
+}
+
+# !! order of packages is important !!
+# packages updated on 5th November 2023
```

Comment:
@lazariv Did you check if updates are available for the R packages, via https://cran.r-project.org/web/packages/available_packages_by_name.html for example?

The external `easy_update` tool may be useful here, see https://github.com/fizwit/easy_update

Reply:
I will do after version 4.3.3 is released